### ISSUE TITLE
how to evaluate the model which is pretrained with your script ?

### ISSUE THREAD

**Issue by Vincent630** [liulj13/py-MDNet-master#1]
i notice that you post lot of result which is as the format of .mat, dose  that mean you trained by matlab?so if we trained the model with pytorch ,how to evaluate the model ?please let me konw if you hava any idea about this ,and thanks you so much

**Comment by liulj13** [comment]
No, this is the python-version of MDNet trained with pytorch, and the results are saved in .mat because the evaluation is done using the official code provided by OTB which is written in matlab. see http://cvlab.hanyang.ac.kr/tracker_benchmark/ for more details~

**Comment by liulj13** [comment]
No, this is the python-version of MDNet trained with pytorch, and the results are saved in .mat because the evaluation is done using the official code provided by OTB which is written in matlab. see http://cvlab.hanyang.ac.kr/tracker_benchmark/ for more details~
